src/components/SettingsDialog.jsx

```diff
@@ -108,7 +108,7 @@ const SettingsDialog = ({
                     { isFavoritable &&
                     <StarIcon
                         onClick={onStarClick}
-                        className={`margin ${stared ? 'ciq-active-favorite' : ''}`}
+                        className={`margin icon-favorite ${stared ? 'ciq-active-favorite' : ''}`}
```

Comment:
our icons are using the convention `ic-{icon name}`. Best to stick to that naming convention; codebase needs to be consistent throughout even though it is written collaboratively.

Reply:
@brucebinary you are right, I follow the market and name it `ciq-favorite`, is this ok?